### DISCUSSION TITLE
[Bug]: Category toggler doesn't show save button correctly

### DISCUSSION THREAD

**Discussion by marnauortega** [swellstores/community#376] [Help] [answered]
Small bugs,

When I disable a category, the save button is shown, and I save. When I enable the category the save button isn't shown, and it should.

https://github.com/swellstores/community/assets/37710726/55a29ea0-f207-4f11-8dbc-ff5dd3e174c0

Less important, when I disable a category and reenable it, the save button is shown, and it needn't, since no change has been made.

https://github.com/swellstores/community/assets/37710726/2bc20cd9-1e56-47c1-b9df-cce2c7749dca

P.S. Is it ok if I report these small things? Or being flooded with minor things keeps you distracted from the important ones?

**Comment by sags95** [comment, marked as answer]
Hey @marnauortega , thanks for reporting this. Will take a look to see if these have been reported already, otherwise will file with the team!
